client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.scss

```diff
@@ -99,7 +103,8 @@
 		}
 
 		&::after {
-			box-shadow: 0 0 0 2px var(--color-primary-light);
+			// Scale up the border of a hovered pattern in fixed viewport
+			box-shadow: 0 0 0 calc(2px * (1 / var(--viewport-scale))) var(--color-primary-light);
```

Comment:
It looks like the scale is not working well. It shouldn't be so thick. Should we use a border instead?

Reply:
I cannot reproduce the issue, have you found steps to reproduce it? The issue seems to be the scale not being updated.

I think it's fine to use shadow rather than border because it works fine when the scale is correct. 

<img width="1348" alt="Screenshot 2566-06-12 at 16 03 50" src="https://github.com/Automattic/wp-calypso/assets/1881481/f45e6ebe-26a7-4041-88fc-60f349630e30">

<img width="846" alt="Screenshot 2566-06-12 at 16 01 45" src="https://github.com/Automattic/wp-calypso/assets/1881481/8f400412-b667-4fa5-9dc9-6680b4872b02">

<img width="518" alt="Screenshot 2566-06-12 at 16 03 57" src="https://github.com/Automattic/wp-calypso/assets/1881481/8a11b5d3-3fcf-4cc6-ba06-328cae43643e">

Reply:
I just reproduce it by making the window smaller, so small that seems to be an edge case but I think I can fix it.

<img width="936" alt="Screenshot 2566-06-12 at 16 16 41" src="https://github.com/Automattic/wp-calypso/assets/1881481/cda01522-b995-4395-8fd4-7edd89d46f42">

Reply:
Using a border fixes it! Is there any reason to use a shadow?

https://github.com/Automattic/wp-calypso/assets/1881481/14a6a51e-f4ef-48a1-923f-928bb72e59c6